spec/fixtures/functions/grid-calc.scss

```diff
@@ -0,0 +1,30 @@
+@import "../setup";
```

Comment:
Same thing here…I think these can be without the relative path?

https://github.com/thoughtbot/bourbon/blob/master/spec/fixtures/library/border-color.scss#L1